addons/optionsmenu/config.cpp

```diff
@@ -32,3 +32,10 @@ class CfgAddons {
 class ACE_Extensions {
     extensions[] += {"ace_clipboard"};
 };
+
+
+class CfgCommands {
+    allowedHTMLLoadURIs[] = {
```

Comment:
Shouldn't it be `+=` (if that works) so `allowedHTMLLoadURIs` from anything else wouldn't get deleted?